src/react/src/components/Contribute/SearchByNameAndAddressTab.jsx

```diff
@@ -51,25 +72,29 @@ const SearchByNameAndAddressTab = ({
     const [addressTouched, setAddressTouched] = useState(false);
     const [countryTouched, setCountryTouched] = useState(false);
 
+    const isCountryError = countryTouched && !inputCountry?.value;
+
     const history = useHistory();
-    const isValid = val => {
-        if (val) {
-            return val.length > 0;
-        }
-        return false;
-    };
+    const isValid = val => val.trim().length > 0;
```

Comment:
Suggestion: use Lodash methods here:

```
import isEmpty from 'lodash/isEmpty';
import trim from 'lodash/trim';

const isValid = val => !isEmpty(trim(val));
```

Reply:
Thanks for your suggestion, but in this particular case I prefer to use the native approach.
If you don't mind I will keep it as is.